docs/pyrona.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Pyrona - Fearless Concurrency for Python
+---
+
+# Pyrona - Fearless Concurrency for Python
+
+As part of Project Verona, we have been developing a new ownership model for Python, called Lungfish.
+This model is designed to provide a safe and efficient way to manage memory and concurrency in Python programs.
+Pyrona is our experiments to develop this approach.
+
+
+## Plan for Python
+
+### Quick prototyping in FrankenScript
+
+Modifying a production language is a complex task.
+As an initial step, we have been developing a toy language called [FrankenScript](https://github.com/fxpl/frankenscript) that allows us to quickly prototype our ideas for region-based ownership.
+FrankenScript is a small language that is designed to be easy to modify and extend.
+It is based on the ideas of ownership and concurrency that we are exploring in Project Verona, but where all the checks are dynamic.
+
+This prototype has given us confidence in the conceptual ideas behind our ownership model, and has allowed us to explore the design space of ownership in a dynamic language.
+
+### Engaging with the Python community
+
+Over the last two years, we have been engaging with the Faster CPython team at Microsoft as a sounding board for our ideas.
+In May 2025, we will be taking our first steps into the boarder Python community at the [Python Language Summit](https://us.pycon.org/2025/events/language-summit/).
+We will be presenting our ideas and seeking feedback from the core developers of the language.
+
+
+### Steps to a new ownership model
+
+Building an ownership model for Python is a complex task, and we are taking a step-by-step approach to ensure that we get it right.
+
+The first step is actually to build a concept of deep immutability into Python.  This can be split into three parts:
+
+* Deep Immutability: We are starting with a deep immutability model, we have been drafting a [PEP](https://github.com/TobiasWrigstad/peps/pull/8) to describe this model.
+
+* SCC work and atomic RC
+
+* Integration with message passing between sub-interpreters
```

Comment:
Should link to PEP734. Maybe say something about what we enable like sharing data by reference, module consistency, and that this makes it possible to get parallel performance even though we are technically not losing the GIL.